### DISCUSSION TITLE
Enhancement: Multispeaker Implementation (for Podcasts or creating "conversations" within audiobooks, etc)

### DISCUSSION THREAD

**Discussion by ldavis9000aws** [erew123/alltalk_tts#440] [General]
I noticed that in the original F5 TTS implementation, you can specify multiple speakers and assign each one a voice. I can do something close to this utilizing the "narrator" approach in AllTalkTTS v2, but that only allows for 2 speakers within the generated conversation. It would be nice to have a "Podcast" type option similar to what was implemented in the PIP module for F5TTS.

**Comment by erew123** [comment]
Hi @ldavis9000aws 

This actually is already in the Features request list https://github.com/erew123/alltalk_tts/discussions/74

![image](https://github.com/user-attachments/assets/ea24b177-3c18-4b32-b07e-fd83eb970c69)

Its just I have a list as long as my arm of things to get done. Im just working on RVC training, which has resulted in an RVC pipeline rebuild and now resulting in adding Openvoice as an alternative pipeline to RVC. Then document etc, then on to something else in the list.... plus obviously I have life stuff going on.

So, it will be something I look at, I just want to get the rest of this code clean-up done and knock off this voice pipeline.
